test/test_workflows.py

```diff
@@ -540,35 +402,6 @@ def test_run_transformation_validation_failure(
         # --- Crypto example ---
         # Complete
         ("test/workflows/crypto/description.cwl", None),
-        # --- Pi example ---
-        # There is no input expected
-        (
-            "test/workflows/pi/description.cwl",
-            "test/workflows/pi/type_dependencies/production/metadata-pi_complete.yaml",
-        ),
```

Comment:
Don't we want to keep at least `Pi`? Because from what I see you didn't keep any `Workflow`, is that expected?

Reply:
Keeping a small number of workflows can help us seeing concretely what we are doing.
Unless I miss something?

Reply:
we do have crypto workflows, so some small tests are still running.